docs/source/distributed_runner.rst

```diff
@@ -19,7 +19,7 @@ Very similar to this, ``vivarium-cluster-tools`` includes a command for simulati
 
     psimulate run /path/to/your/model/specification  /path/to/your/branch
 
-By default, output will be saved in ``/share/costeffectiveness/results``. If you want to save the
+By default, output will be saved in ``/mnt/team/simulation_science/costeffectiveness/results``. If you want to save the
```

Comment:
Is this still the right default? `/mnt/team/simulation_science/priv/engineering/` or a subdirectory might be better.